src/pseudopeople/interface.py

```diff
@@ -152,9 +152,11 @@ def generate_decennial_census(
 
     :param source: The root directory containing pseudopeople simulated population data. Defaults
         to using the included sample population when source is `None`.
-    :param seed: An integer seed for randomness.
+    :param seed: An integer seed for randomness. Defaults to 0.
     :param config: An optional override to the default configuration. Can be a path
-        to a configuration YAML file or a dictionary.
+        to a configuration YAML file, a configuration dictionary,
+        or the sentinel value `pseudopeople.NO_NOISE`, which will generate a
+        dataset with all noise levels set to 0.
```

Comment:
```suggestion
        dataset without any configurable noise.
```

Maybe? I suggest syncing this up with what it says on the Configuration page of the docs; if you prefer what you have here, you might want to edit it there. I think "noise level" is not a phrase we've introduced anywhere -- it only appears in the get_config docstring -- but maybe it is self-explanatory?

Reply:
Good call.